django_celery_beat/models.py

```diff
@@ -8,8 +8,15 @@
 
 import timezone_field
 from celery import current_app, schedules
-from cron_descriptor import (FormatException, MissingFieldException,
-                             WrongArgumentException, get_description)
+
+try:
```

Comment:
We need a comment here explaining why.

Or perhaps in #935 we should just pin the dependency to >= 2.0.